tests/test_utils.py

```diff
@@ -180,6 +180,14 @@ def test_is_valid_url(self):
         assert pipenv.utils.is_valid_url(url)
         assert pipenv.utils.is_valid_url(not_url) is False
 
+    @pytest.mark.parametrize('input_path, expected', [
+        ('artifacts/file.zip', os.path.join('.', 'artifacts', 'file.zip')),
+        ('./artifacts/file.zip', os.path.join('.', 'artifacts', 'file.zip')),
+        ('../otherproject/file.zip', os.path.join('.', '..', 'otherproject', 'file.zip'))
+    ])
+    def test_get_converted_relative_path(self, input_path, expected):
```

Comment:
So I'm a little unsure about this test. It's generally good to have concrete expected values, otherwise we're only ensuring that `os.path.join` is just a broken in `get_converted_relative_path` as it is on it's own.

Reply:
Yeah I agree,  I was fishing for a solution that would accommodate windows which had a special caveat for python 2.7.  I'm just going to write a separate one for windows.